webpack/JobWizard/JobWizardConstants.js

```diff
@@ -14,3 +14,11 @@ export const repeatTypes = {
   daily: __('Daily'),
   hourly: __('Hourly'),
 };
+
+export const wizardTitles = {
```

Comment:
as per https://stackoverflow.com/questions/40291766/naming-convention-for-const-object-keys-in-es6/40291962

constants, sorry... CONSTANTS, should be all capitalized